msu/utils/globals.nut

```diff
@@ -176,11 +176,11 @@
 			}
 			return true;
 		case "instance":
-			if (!(_b instanceof _a.getclass()))
+			if (!(typeof _b != "instance") || _a.getclass() != _b.getclass())
 				return false;
-			foreach (k, v in _a.getclass())
+			foreach (k, v in _a)
```

Comment:
why is this no longer getclass? afaik this has to be getclass otherwise the instances `_next` metamethod is used.

Reply:
I see. I didn't account for the `_nexti` thing. Does the `_nexti` overwrite not apply to iteration on classes?

Reply:
No

Reply:
Ok I'll change it to `getClass()`